libafl_qemu/src/modules/usermode/snapshot.rs

```diff
@@ -796,27 +810,37 @@ where
         )))]
         SYS_newfstatat => {
             if a2 != 0 {
-                let h = hooks.match_helper_mut::<QemuSnapshotHelper>().unwrap();
+                let h = emulator_modules
+                    .match_module_mut::<SnapshotModule>()
```

Comment:
`get_mut`? It's always going to be a module, right?

Reply:
i'd still keep the match part since it will return the first match of the module tuple

Reply:
nevermind, we would have to use `match()`, which is a reserved keyword